hw/ip_templates/gpio/dv/sva/gpio_bind.sv.tpl

```diff
@@ -20,4 +20,14 @@ module ${module_instance_name}_bind;
     .d2h    (tl_o)
   );
 
+  bind ${module_instance_name} ${module_instance_name}_strap_check #(.NUM_GPIOS($bits(cio_${module_instance_name}_i)))
```

Comment:
I'd be inclined to get rid of all the `${module_instance_name}` instances and turn them into just `gpio`. I think we then get rid of the need for templating, so this file can drop its `.tpl` suffix.

I'd suggest you add a commit before this one that makes that change in the existing version of this file.